wicket-core/src/main/java/org/apache/wicket/markup/head/ResourceAggregator.java

```diff
@@ -353,6 +353,7 @@ private void renderCombinedEventScripts()
 		}
 		if (combinedScript.length() > 0)
 		{
+			combinedScript.append("\nWicket.Event.publish(Wicket.Event.Topic.AJAX_TIMERS_BOUND);");
```

Comment:
Right!
The names are confusing!
They should use `load` and `domready` in their names to be more clear.
What about `Wicket.Event.Topic.ONDOMREADY_EVENTS_BOUND` and `...ONLOAD_EVENTS_BOUND` ?